### ISSUE TITLE
ux: `ua disable`

### ISSUE THREAD

**Issue by powersj** [CanonicalLtd/ubuntu-advantage-client#726]
Disable esm-infra:
```
$ sudo ua disable esm-infra
Updating package lists
```

Disable livepatch:
```
$ sudo ua disable livepatch
Removing canonical-livepatch snap
```

**Comment by OddBloke** [comment]
@powersj In the UA UX doc, we seem to suggest that FIPS should be disable-able. Is this an intentional change from the current behaviour (of refusing to disable it at all)?

**Comment by powersj** [comment]
FIPS work will come in the next release not this one, but yes we will work with the team to figure out what packages to remove.

**Comment by OddBloke** [comment]
@powersj At the moment, we don't check people's contracts on disable (because it doesn't really matter if they're entitled to something they are turning off, I suppose). What's the thinking behind doing so?

**Comment by powersj** [comment]
What does checking during disable get us? Is it complexity to the code since we only do it during refresh/enable?

**Comment by powersj** [comment]
oh.. I just looked at the UX and realized it shows us refreshing *facepalm*

I'm fine with removing that from the UX as we don't do it today and I can't come up with a reason for why we would start.

**Comment by powersj** [comment]
![facepalm](https://user-images.githubusercontent.com/6453401/65784059-1b903600-e106-11e9-938a-ca11dc2d8a5d.gif)

**Comment by OddBloke** [comment]
Cool, thanks! Have updated the description to match.

**Comment by OddBloke** [comment]
@powersj OK, I just want to double check what we're doing here. Currently the UX for repo entitlements is:

```
$ sudo ua disable esm-infra
ESM Infra disabled.
```

And we're looking to change it to:

```
$ sudo ua disable esm-infra
Updating package lists
```

That isn't a message that we currently emit, and actually isn't something we do on disable currently. (This is less noticeable than it will be on xenial, because the lists as seen by apt don't change when ESM gets enabled or disabled, so that the MOTD messaging can still function.) Should we start doing an `apt-get update` on disable, too? (Do we want a separate issue for that?)

(For livepatch, we're already emitting the required message, so it's just a case of dropping the `<foo> disabled.` line.)

**Comment by powersj** [comment]
Yes we want to start doing the update and remove the message of disabled
for ESM as well as livepatch.

**Comment by OddBloke** [comment]
This was fixed by #769 and #770.
